src/lib/mina_base/zkapp_call_forest_base.ml

```diff
@@ -350,7 +351,7 @@ module Shape = struct
   type t = Node of (I.t * t) list [@@deriving quickcheck]
 end
 
-let rec shape (t : _ t) : Shape.t =
+let rec shape (t : ('a, 'b, 'c) t) : Shape.t =
```

Comment:
Do you mind add some different type param names here instead of `('a, 'b, 'c)`? Thanks :)